src/lavinmq/server.cr

```diff
@@ -385,19 +385,27 @@ module LavinMQ
     getter stats_system_collection_duration_seconds = Time::Span.new
 
     private def control_flow!
-      if @disk_free < 2_i64 * Config.instance.segment_size || @disk_free < Config.instance.free_disk_min
+      if disk_full?
         if flow?
           @log.info { "Low disk space: #{@disk_free.humanize}B, stopping flow" }
           flow(false)
         end
       elsif !flow?
```

Comment:
Still think the naming is a bit confusing here.. if we don't have flow (flow is stopped because we have a high disk usage?) we want to log "Not low..." ?

Reply:
I guess we could make it more explicit

the `elsif !flow?` implicitly means `if !disk_full? && !flow?` because of the if above. So if disk is not full but flow is set to false, that means disk usage has been reduced below the limit, and flow should be resumed.

Reply:
makes sense :)